docs/serving/using-auto-tls.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Enabling automatic TLS certificate provisioning"
+linkTitle: "Enabling auto TLS certs"
+weight: 64
+type: "docs"
+---
+
+If you install and configure cert-manager, you can configure Knative to
+automatically obtain new TLS certificates and renew existing ones.
+To learn more about using secure connections in Knative, see
+[Configuring HTTPS with TLS certificates](./using-tls-cert.md).
+
+## Before you begin
+
+You must meet the following prerequisites to enable automatic certificate
+provisioning:
+
+- The following must be installed on your Knative cluter:
+  - [Knative Serving version 0.6.0 or higher](../install/).
+  - [Istio with SDS, version 1.1 or higher](../install/installing-istio.md#installing-istio-with-SDS-to-secure-the-ingress-gateway).
+    Note: Currently, [Gloo](https://github.com/solo-io/gloo) is unsupported.
+  - [cert-manager version `0.6.1` or higher](./installing-cert-manager.md).
+- Your Knative cluster must be configured to use a
+  [custom domain](./using-a-custom-domain.md).
+
+
+## Enabling automatic certificate provisioning
+
+To enable Knative to automatically provision TLS certificates:
+
+1. Run the following commands to install the `networking-certmanager`:
```

Comment:
As `networking-certmanager` is installed by default, users may already have `networking-certmanager` installed. So I would rephrase this part as 
```
Run the following commands to install the `networking-certmanager` if it has not been installed in the cluster.
```
or something similar.

Reply:
Is there a command to check if it exists?

I can say run this command:

kubectl get.....????? networking-certmanager

If you get "DOES NOT EXIST" (not sure what the response would be), then run the following command:

Reply:
I updated the doc to use `kubectl get` to check if networking-certmanager exists first.

Reply:
thanks!